backend/dataall/modules/dataset_sharing/aws/lakeformation_client.py

```diff
@@ -254,46 +254,48 @@ def revoke_source_table_access(**data):
         source_table = data['source_table']
         source_accountid = data['source_accountid']
         for target_principal in target_principals:
-            try:
```

Comment:
I am not a huge fan of handling the QS Principals - want to think on other ways to do this 

In theory I would think the permissions for the target QS Group on the source database/table would only get removed when the external access to the target Aws Account is removed (i.e. when `revoke_external_account_access_on_source_account()` is called)

Will have to test though...